lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -565,7 +600,7 @@ def server_change_attributes(compute_api, target_server, wished_server):
 def core(module):
     region = module.params["region"]
     wished_server = {
-        "state": module.params["state"],
+        "state": "absent" if module.params["state"] == "terminated" else module.params["state"],
```

Comment:
@g1franc I'm not sure about this line. Why is it here?

Reply:
It use the value received from the state argument. As it was changed to "terminated", this condition has to be updated too.